index.html

```diff
@@ -5485,13 +5532,62 @@ <h5>Conclusion</h5>
 See examples.
 -->
 <p>
-Based on the limited data, we are <a data-ucr-role="conclusion">undecided</a> about whether this should be a requirement.
+This capability is a <a data-ucr-role="conclusion">requirement</a> for the scripting API of a native HTML map element.
+Responding to changes in the state of a map, or user interactions with elements of the map such as markers,
+is necessary for a web application to usefully modify its own state so as to reflect the results of such interactions.
 </p>
 <ul data-ucr-role="conclusion-notes">
 <!-- bullet-point notes summarizing implementation details / blocking issues.
 Add tags to the start of a point with an empty `<a data-ucr-role="tag"></a>` where the text content or href matches a tag `id` value. -->
-<li><a data-ucr-role="tag" href=""></a>
-<!-- short summary of how the tag applies -->
+<li><a data-ucr-role="tag">privacy-fingerprinting</a>
+As scripts may be loaded from arbitrary origins (depending on cross-rigin restrictions),
```

Comment:
Typo "cross-rigin"

Reply:
Oops! Fixed, thanks :-)

…though I've just realised WebStorm has somehow switched itself to my other GitHub account for that commit.